R/Backend-class.R

```diff
@@ -112,9 +114,16 @@ setClass("Backend",
     NULL
 }
 
+.valid.Backend.processingQueue <- function(x) {
+    if (length(x))
+        if (!all(vapply(x, inherits, logical(1), "ProcessingStep")))
+            return("'processingQueue' should only contain ProcessingStep objects.")
+    NULL
+}
+
 setValidity("Backend", function(object) {
     msg <- .valid.Backend.files(object@files)
-
+    msg <- c(msg, .valid.Backend.processingQueue(object@processingQueue))
```

Comment:
Just use:
```r
msg <- c(
    .valid.Backend.files(object@files),
    .valid.Backend.processingQueue(object@processingQueue)
)
```
instead of 
```r
msg <- c(msg, ...)
msg <- c(msg, ...)
```

Reply:
👍

Reply:
No need for this in the new code.